src/prefect/server/utilities/messaging/__init__.py

```diff
@@ -131,14 +131,21 @@ class CacheModule(Protocol):
     Cache: Type[Cache]
 
 
-def create_cache() -> Cache:
+def create_cache(topic: Optional[str] = None) -> Cache:
```

Comment:
I think we should hold off on adding this kwarg for now. There isn't a convenient way for users to set this right now, and I think we can get away with a sane default until users request a custom topic for the cache.